### ISSUE TITLE
Template for new translations => Unrecognized base file for new translations

### ISSUE THREAD

**Issue by myii** [WeblateOrg/weblate#4828]
**Describe the bug**

After scanning my repo (https://gitlab.com/myii/ut-tweak-tool), it correctly finds the `.pot` file (`po/ut-tweak-tool.sverzegnassi.pot`) but then results in this error when attempting to save:

```
Unrecognized base file for new translations.

Filename of file used for creating new translations. For gettext choose .pot file.
```

* I'm wondering whether it's having trouble with the second period (`.`).

**To Reproduce the bug**

Description should look similar to this:

As described above.

**Expected behavior**

`po/ut-tweak-tool.sverzegnassi.pot` should be accepted as a valid `.pot` file.

**Screenshots**

N/a.

**Server configuration and status**

**Weblate deploy checks**

**Exception traceback**

**Additional context**

**Comment by nijel** [comment]
It fails with following error:

> Syntax error on line 1073

But Weblate should report this to user instead of just showing it in the logs...

**Comment by myii** [comment]
@nijel I still don't see any further error messages when attempting to save.  In any case, the bug I've reported here is still unresolved:

* `po/ut-tweak-tool.sverzegnassi.pot` should be accepted as a valid `.pot` file.

**Comment by nijel** [comment]
> should be accepted as a valid .pot file.

No, because it isn't - the entries should be separated by whitespace lines, but the obsolete entries in your file aren't. See https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html for syntax documentation.

What was fixed is displaying of the error message. 

> I still don't see any further error messages when attempting to save. 

It was fixed in git and is not yet part of any release. In case you are using Hosted Weblate service (it would be nice to mention that), it was not yet deployed.

**Comment by myii** [comment]
> > should be accepted as a valid .pot file.
> 
> No, because it isn't - the entries should be separated by whitespace lines, but the obsolete entries in your file aren't. See https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html for syntax documentation.

Thanks, corrected this and now it's working (hosted Weblate service).  I ended up using [Dennis](https://github.com/willkg/dennis) to confirm I'd cleared all of the violations.  I was going to suggest mentioning that somewhere for end users but then I saw the conversation at #1252.  I suppose the fix 1435729 will provide the lint error, so won't be an issue when that drops.
